SasquatchMac/SasquatchMac/ViewControllers/CrashesViewController.swift

```diff
@@ -29,6 +31,7 @@ class CrashesViewController : NSViewController, MobileCenterProtocol {
     guard let `mobileCenter` = mobileCenter else {
       return
     }
+    ServiceStateStore.CrashesState = sender.state == 1
```

Comment:
We don't need this part I think. below line will update the value to storage since we are referencing the same key.